medusa/server/web/config/notifications.py

```diff
@@ -251,6 +253,13 @@ def saveNotifications(self, use_kodi=None, kodi_always_on=None, kodi_notify_onsn
         app.PUSHBULLET_API = pushbullet_api
         app.PUSHBULLET_DEVICE = pushbullet_device_list
 
+        app.USE_JOIN = config.checkbox_to_value(use_join)
```

Comment:
These are actually not needed anymore. Saving is now going via apiv2.

But you can leave them like is for now. Well clean them up later.

Reply:
Should I remove the whole app.*JOIN section or just the one line you referenced?

Reply:
Leave them for now

Reply:
ok, will do